pkg/ansible/controller/reconcile.go

```diff
@@ -183,7 +182,20 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 			}
 		}
 		if event.Event == eventapi.EventRunnerOnFailed && !event.IgnoreError() {
+			eventErr := errors.New("event runner on failed")
+			failureMessages := eventapi.FailureMessages{}
```

Comment:
I am pretty sure this will eat ignored failures

Reply:
Hi @shamsher31 and @fabianvf, 
 
Should we mark all errors and just retrigger the reconcile if `!event.IgnoreError()` ? Is that the expected behaviour?

Reply:
I think we shouldn't break the loop here actually, we should wait until all events are processed before we determine whether the run is successful or failed. We can probably just add a check after we determine if the run is successful, and add `Requeue: True` to reconcileResult if it was not. We would just need to add this line back and add a check right after, since all the returns after that return `reconcileResult` I think that should do it https://github.com/operator-framework/operator-sdk/pull/2375/files#diff-9649888b3df1961dad649345545de4c0L215

Reply:
Make sense. Now I got why it was using the msg var before. I will change it now.